test/mocks/mocks.go

```diff
@@ -360,8 +360,9 @@ func SecretMock(ns, name string) *corev1.Secret {
 			APIVersion: "v1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Namespace: ns,
-			Name:      name,
+			Namespace:       ns,
+			Name:            name,
+			ResourceVersion: "116076",
```

Comment:
This line is causing test failure because resourceVersion is set by cluster while creating an object, I think we can not set it manually. The `get, list and watch operations` support the resourceVersion parameter.
```
 TestAddSchemesToFramework/end-to-end/scenario-db-app-sbr: servicebindingrequest_test.go:308: Creating Database credentials secret mock object...
    TestAddSchemesToFramework/end-to-end/scenario-db-app-sbr: servicebindingrequest_test.go:310: 
        	Error Trace:	servicebindingrequest_test.go:310
        	            				servicebindingrequest_test.go:452
        	            				servicebindingrequest_test.go:155
        	            				servicebindingrequest_test.go:91
        	Error:      	Received unexpected error:
        	            	resourceVersion should not be set on objects to be created
```